code/game/objects/effects/effect_system.dm

```diff
@@ -94,31 +94,31 @@ steam.start() -- spawns the effect
 // will always spawn at the items location.
 /////////////////////////////////////////////
 
-/obj/effect/sparks

```

Comment:
Because obj/effect/effect is actually a used subsystem:
```
/obj/effect/effect
	name = "effect"
	icon = 'icons/effects/effects.dmi'
	mouse_opacity = 0
	unacidable = 1//So effect are not targeted by alien acid.
	pass_flags = PASSTABLE | PASSGRILLE
```
👍 

Blame oldcode. It could probably be renamed to like, obj/effect/particles or some such. /shrug.

Also, @Alberyk , the issue itself could probably be fixed if you add the line `	icon = 'icons/effects/effects.dmi'` instead of adding the parent class. Maybe. But this works too.